src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1735,10 +1813,13 @@ SubCmdValidate::execute(const SubCmdOptions& _options) const
 
     // Logic for individually defined tests
     // Enqueue the matching test suites to be executed
-    std::string testSuiteName = get_test_name(testSuite[index].ptTest.get("name","<unknown>"));
     for (const auto & testName : testsToRun) {
       if (testName.compare(testSuiteName) == 0) {
```

Comment:
```suggestion
      if (boost::equals(testName, testSuiteName)) {
```
Might as well fix this one too!